lib/plugins/wallet-scoring/scorechain/scorechain.js

```diff
@@ -0,0 +1,74 @@
+const axios = require('axios')
+const _ = require('lodash/fp')
+
+
+const NAME = 'Scorechain'
+const SUPPORTED_COINS = {
+  BTC: 'BITCOIN',
+  ETH: 'ETHEREUM',
+  USDT: 'ETHEREUM',
+  BCH: 'BITCOINCASH',
+  LTC: 'LITECOIN',
+  DASH: 'DASH',
+  TRX: 'TRON',
+  USDT_TRON: 'TRON'
+}
+
+const TYPE = {
+  TRANSACTION: 'TRANSACTION',
+  ADDRESS: 'ADDRESS'
+}
+
+function rate (account, objectType, cryptoCode, objectId) {
+  if (_.isNil(account) || !account.enabled || !Object.keys(SUPPORTED_COINS).includes(cryptoCode)) return Promise.resolve(null)
```

Comment:
This could be replaced by `isWalletScoringEnabled(account, cryptoCode)`

Reply:
Nice find.